src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const totalCost = 40 * days;
```

Comment:
It would be good to have `40` as a variable with descriptive name as well, so that we know what it means